.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  tabWidth: 2,
+  semi: true,
+  singleQuote: true,
+  useTabs: false,
+  jsxSingleQuote: false,
+  trailingComma: "all",
+  printWidth: 120,
+  quoteProps: "as-needed",
+  bracketSpacing: true,
+  jsxBracketSameLine: false,
+  arrowParens: "always",
+  parser: "typescript",
+  endOfLine: "crlf",
```

Comment:
I think we shouldn't change the default setting here. In most cases, this code will run on Linux machines so "lf" could be better